From 1683c3a7e654f4d5d4b6863f675deee2761f3394 Mon Sep 17 00:00:00 2001
From: Ryan Blue <blue@apache.org>
Date: Thu, 14 May 2015 15:39:39 -0700
Subject: [PATCH 113/176] PARQUET-265: Update POM files for Parquet TLP.

Author: Ryan Blue <blue@apache.org>

Closes #186 from rdblue/PARQUET-265-update-build-for-graduation and squashes the following commits:

7bd2931 [Ryan Blue] PARQUET-265: Update POM files for Parquet TLP.

Conflicts:
	dev/source-release.sh
	parquet-benchmarks/pom.xml
	pom.xml
Resolution:
    source-release script is not backported
    minor conflicts in other poms, using upstream values.
---
 parquet-avro/pom.xml                               |    4 ++--
 parquet-benchmarks/pom.xml                         |    4 ++--
 parquet-cascading/pom.xml                          |    4 ++--
 parquet-column/pom.xml                             |    4 ++--
 parquet-common/pom.xml                             |    4 ++--
 parquet-encoding/pom.xml                           |    4 ++--
 parquet-generator/pom.xml                          |    4 ++--
 parquet-hadoop-bundle/pom.xml                      |    4 ++--
 parquet-hadoop/pom.xml                             |    4 ++--
 parquet-hive-bundle/pom.xml                        |    4 ++--
 .../parquet-hive-0.10-binding/pom.xml              |    2 +-
 .../parquet-hive-0.12-binding/pom.xml              |    2 +-
 .../parquet-hive-binding-bundle/pom.xml            |    2 +-
 .../parquet-hive-binding-factory/pom.xml           |    2 +-
 .../parquet-hive-binding-interface/pom.xml         |    2 +-
 parquet-hive/parquet-hive-binding/pom.xml          |    2 +-
 parquet-hive/parquet-hive-storage-handler/pom.xml  |    2 +-
 parquet-hive/pom.xml                               |    4 ++--
 parquet-jackson/pom.xml                            |    4 ++--
 parquet-pig-bundle/pom.xml                         |    4 ++--
 parquet-pig/pom.xml                                |    4 ++--
 parquet-protobuf/pom.xml                           |    4 ++--
 parquet-scala/pom.xml                              |    4 ++--
 parquet-scrooge/pom.xml                            |    4 ++--
 parquet-thrift/pom.xml                             |    4 ++--
 parquet-tools/pom.xml                              |    4 ++--
 pom.xml                                            |   22 ++++++++++----------
 27 files changed, 56 insertions(+), 56 deletions(-)

diff --git a/parquet-avro/pom.xml b/parquet-avro/pom.xml
index e999f9f..de93aeb 100644
--- a/parquet-avro/pom.xml
+++ b/parquet-avro/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-avro</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Avro (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Avro</name>
+  <url>https://parquet.apache.org</url>
 
   <properties>
   </properties>
diff --git a/parquet-benchmarks/pom.xml b/parquet-benchmarks/pom.xml
index 821a79b..60e061e 100644
--- a/parquet-benchmarks/pom.xml
+++ b/parquet-benchmarks/pom.xml
@@ -28,8 +28,8 @@
 
   <artifactId>parquet-benchmarks</artifactId>
   <packaging>jar</packaging>
-  <name>Parquet Benchmarks</name>
-  <url>https://github.com/Parquet/parquet-mr</url>
+  <name>Apache Parquet Benchmarks</name>
+  <url>https://parquet.apache.org</url>
 
   <properties>
     <jmh.version>1.3.4</jmh.version>
diff --git a/parquet-cascading/pom.xml b/parquet-cascading/pom.xml
index a091505..254f656 100644
--- a/parquet-cascading/pom.xml
+++ b/parquet-cascading/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-cascading</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Cascading (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Cascading</name>
+  <url>https://parquet.apache.org</url>
 
   <repositories>
     <repository>
diff --git a/parquet-column/pom.xml b/parquet-column/pom.xml
index 92ce6bb..41992b3 100644
--- a/parquet-column/pom.xml
+++ b/parquet-column/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-column</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Column (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Column</name>
+  <url>https://parquet.apache.org</url>
 
   <properties>
   </properties>
diff --git a/parquet-common/pom.xml b/parquet-common/pom.xml
index 88707ce..50e1c22 100644
--- a/parquet-common/pom.xml
+++ b/parquet-common/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-common</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Common (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Common</name>
+  <url>https://parquet.apache.org</url>
 
   <properties>
   </properties>
diff --git a/parquet-encoding/pom.xml b/parquet-encoding/pom.xml
index 496a6fc..d6e2539 100644
--- a/parquet-encoding/pom.xml
+++ b/parquet-encoding/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-encoding</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Encodings (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Encodings</name>
+  <url>https://parquet.apache.org</url>
 
   <properties>
   </properties>
diff --git a/parquet-generator/pom.xml b/parquet-generator/pom.xml
index 333821c..1ec5f9f 100644
--- a/parquet-generator/pom.xml
+++ b/parquet-generator/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-generator</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Generator (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Generator</name>
+  <url>https://parquet.apache.org</url>
 
   <properties>
   </properties>
diff --git a/parquet-hadoop-bundle/pom.xml b/parquet-hadoop-bundle/pom.xml
index 6bcd933..9964e76 100644
--- a/parquet-hadoop-bundle/pom.xml
+++ b/parquet-hadoop-bundle/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-hadoop-bundle</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Hadoop Bundle (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Hadoop Bundle</name>
+  <url>https://parquet.apache.org</url>
 
   <properties>
   </properties>
diff --git a/parquet-hadoop/pom.xml b/parquet-hadoop/pom.xml
index 4cf9fb8..f8808bc 100644
--- a/parquet-hadoop/pom.xml
+++ b/parquet-hadoop/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-hadoop</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Hadoop (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Hadoop</name>
+  <url>https://parquet.apache.org</url>
 
   <properties>
   </properties>
diff --git a/parquet-hive-bundle/pom.xml b/parquet-hive-bundle/pom.xml
index b1a26fe..1812c13 100644
--- a/parquet-hive-bundle/pom.xml
+++ b/parquet-hive-bundle/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-hive-bundle</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Hive Bundle (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Hive Bundle</name>
+  <url>https://parquet.apache.org</url>
 
   <properties>
   </properties>
diff --git a/parquet-hive/parquet-hive-binding/parquet-hive-0.10-binding/pom.xml b/parquet-hive/parquet-hive-binding/parquet-hive-0.10-binding/pom.xml
index 9847cf4..de1e4de 100644
--- a/parquet-hive/parquet-hive-binding/parquet-hive-0.10-binding/pom.xml
+++ b/parquet-hive/parquet-hive-binding/parquet-hive-0.10-binding/pom.xml
@@ -27,7 +27,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>parquet-hive-0.10-binding</artifactId>
-  <name>Apache Parquet Hive 0.10 Binding (Incubating)</name>
+  <name>Apache Parquet Hive 0.10 Binding</name>
   <packaging>jar</packaging>
 
   <properties>
diff --git a/parquet-hive/parquet-hive-binding/parquet-hive-0.12-binding/pom.xml b/parquet-hive/parquet-hive-binding/parquet-hive-0.12-binding/pom.xml
index 671c77c..5b6aa38 100644
--- a/parquet-hive/parquet-hive-binding/parquet-hive-0.12-binding/pom.xml
+++ b/parquet-hive/parquet-hive-binding/parquet-hive-0.12-binding/pom.xml
@@ -27,7 +27,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>parquet-hive-0.12-binding</artifactId>
-  <name>Apache Parquet Hive 0.12 Binding (Incubating)</name>
+  <name>Apache Parquet Hive 0.12 Binding</name>
   <packaging>jar</packaging>
 
   <properties>
diff --git a/parquet-hive/parquet-hive-binding/parquet-hive-binding-bundle/pom.xml b/parquet-hive/parquet-hive-binding/parquet-hive-binding-bundle/pom.xml
index 2907afe..736439c 100644
--- a/parquet-hive/parquet-hive-binding/parquet-hive-binding-bundle/pom.xml
+++ b/parquet-hive/parquet-hive-binding/parquet-hive-binding-bundle/pom.xml
@@ -29,7 +29,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>parquet-hive-binding-bundle</artifactId>
-  <name>Apache Parquet Hive Binding Bundle (Incubating)</name>
+  <name>Apache Parquet Hive Binding Bundle</name>
   <packaging>jar</packaging>
 
   <dependencies>
diff --git a/parquet-hive/parquet-hive-binding/parquet-hive-binding-factory/pom.xml b/parquet-hive/parquet-hive-binding/parquet-hive-binding-factory/pom.xml
index 57c3a60..06c25dd 100644
--- a/parquet-hive/parquet-hive-binding/parquet-hive-binding-factory/pom.xml
+++ b/parquet-hive/parquet-hive-binding/parquet-hive-binding-factory/pom.xml
@@ -27,7 +27,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>parquet-hive-binding-factory</artifactId>
-  <name>Apache Parquet Hive Binding Factory (Incubating)</name>
+  <name>Apache Parquet Hive Binding Factory</name>
   <packaging>jar</packaging>
 
   <dependencies>
diff --git a/parquet-hive/parquet-hive-binding/parquet-hive-binding-interface/pom.xml b/parquet-hive/parquet-hive-binding/parquet-hive-binding-interface/pom.xml
index c1fd1e3..003ccd6 100644
--- a/parquet-hive/parquet-hive-binding/parquet-hive-binding-interface/pom.xml
+++ b/parquet-hive/parquet-hive-binding/parquet-hive-binding-interface/pom.xml
@@ -27,7 +27,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>parquet-hive-binding-interface</artifactId>
-  <name>Apache Parquet Hive Binding Interface (Incubating)</name>
+  <name>Apache Parquet Hive Binding Interface</name>
   <packaging>jar</packaging>
 
   <dependencies>
diff --git a/parquet-hive/parquet-hive-binding/pom.xml b/parquet-hive/parquet-hive-binding/pom.xml
index 6f89b52..2ba8811 100644
--- a/parquet-hive/parquet-hive-binding/pom.xml
+++ b/parquet-hive/parquet-hive-binding/pom.xml
@@ -27,7 +27,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>parquet-hive-binding</artifactId>
-  <name>Apache Parquet Hive Binding Parent (Incubating)</name>
+  <name>Apache Parquet Hive Binding Parent</name>
   <packaging>pom</packaging>
 
   <!-- provides javax.jdo:jdo2-api:jar:2.3-ec -->
diff --git a/parquet-hive/parquet-hive-storage-handler/pom.xml b/parquet-hive/parquet-hive-storage-handler/pom.xml
index 6b57a37..fc43248 100644
--- a/parquet-hive/parquet-hive-storage-handler/pom.xml
+++ b/parquet-hive/parquet-hive-storage-handler/pom.xml
@@ -27,7 +27,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>parquet-hive-storage-handler</artifactId>
-  <name>Parquet Hive Storage Handler</name>
+  <name>Apache Parquet Hive Storage Handler</name>
   <packaging>jar</packaging>
 
   <repositories>
diff --git a/parquet-hive/pom.xml b/parquet-hive/pom.xml
index dcb9566..05b2905 100644
--- a/parquet-hive/pom.xml
+++ b/parquet-hive/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-hive</artifactId>
   <packaging>pom</packaging>
 
-  <name>Apache Parquet Hive (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Hive</name>
+  <url>https://parquet.apache.org</url>
 
   <developers>
     <developer>
diff --git a/parquet-jackson/pom.xml b/parquet-jackson/pom.xml
index 56d7b42..c7cb3aa 100644
--- a/parquet-jackson/pom.xml
+++ b/parquet-jackson/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-jackson</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Jackson (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Jackson</name>
+  <url>https://parquet.apache.org</url>
 
   <dependencies>
     <dependency>
diff --git a/parquet-pig-bundle/pom.xml b/parquet-pig-bundle/pom.xml
index 4cf9290..4a50032 100644
--- a/parquet-pig-bundle/pom.xml
+++ b/parquet-pig-bundle/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-pig-bundle</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Pig Bundle (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Pig Bundle</name>
+  <url>https://parquet.apache.org</url>
 
   <properties>
   </properties>
diff --git a/parquet-pig/pom.xml b/parquet-pig/pom.xml
index f103657..93ea33a 100644
--- a/parquet-pig/pom.xml
+++ b/parquet-pig/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-pig</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Pig (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Pig</name>
+  <url>https://parquet.apache.org</url>
 
   <properties>
   </properties>
diff --git a/parquet-protobuf/pom.xml b/parquet-protobuf/pom.xml
index 00ee238..e1afd19 100644
--- a/parquet-protobuf/pom.xml
+++ b/parquet-protobuf/pom.xml
@@ -34,8 +34,8 @@
   </properties>
 
 
-  <name>Apache Parquet Protobuf (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Protobuf</name>
+  <url>https://parquet.apache.org</url>
 
   <dependencies>
     <dependency>
diff --git a/parquet-scala/pom.xml b/parquet-scala/pom.xml
index ccd588c..c5ff595 100644
--- a/parquet-scala/pom.xml
+++ b/parquet-scala/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-scala_${scala.binary.version}</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Scala (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Scala</name>
+  <url>https://parquet.apache.org</url>
 
   <repositories>
     <repository>
diff --git a/parquet-scrooge/pom.xml b/parquet-scrooge/pom.xml
index 4649dd2..5a8bf09 100644
--- a/parquet-scrooge/pom.xml
+++ b/parquet-scrooge/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-scrooge_${scala.binary.version}</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Scrooge (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Scrooge</name>
+  <url>https://parquet.apache.org</url>
 
   <repositories>
     <repository>
diff --git a/parquet-thrift/pom.xml b/parquet-thrift/pom.xml
index 373e18e..8cb53d6 100644
--- a/parquet-thrift/pom.xml
+++ b/parquet-thrift/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-thrift</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Thrift (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Thrift</name>
+  <url>https://parquet.apache.org</url>
 
   <properties>
     <elephant-bird.version>4.4</elephant-bird.version>
diff --git a/parquet-tools/pom.xml b/parquet-tools/pom.xml
index a28f6b1..581c27b 100644
--- a/parquet-tools/pom.xml
+++ b/parquet-tools/pom.xml
@@ -29,8 +29,8 @@
   <artifactId>parquet-tools</artifactId>
   <packaging>jar</packaging>
 
-  <name>Apache Parquet Tools (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet Tools</name>
+  <url>https://parquet.apache.org</url>
 
   <properties>
   </properties>
diff --git a/pom.xml b/pom.xml
index df69319..7529b45 100644
--- a/pom.xml
+++ b/pom.xml
@@ -31,14 +31,14 @@
   <version>1.5.0-cdh5.5.0-SNAPSHOT</version>
   <packaging>pom</packaging>
 
-  <name>Apache Parquet MR (Incubating)</name>
-  <url>https://parquet.incubator.apache.org</url>
+  <name>Apache Parquet MR</name>
+  <url>https://parquet.apache.org</url>
   <description>Parquet is a columnar storage format that supports nested data. This provides the java implementation.</description>
 
   <scm>
-    <connection>scm:git:git@github.com:apache/incubator-parquet-mr.git</connection>
-    <url>scm:git:git@github.com:apache/incubator-parquet-mr.git</url>
-    <developerConnection>scm:git:https://git-wip-us.apache.org/repos/asf/incubator-parquet-format.git</developerConnection>
+    <connection>scm:git:git@github.com:apache/parquet-mr.git</connection>
+    <url>scm:git:git@github.com:apache/parquet-mr.git</url>
+    <developerConnection>scm:git:https://git-wip-us.apache.org/repos/asf/parquet-mr.git</developerConnection>
   </scm>
 
   <licenses>
@@ -56,15 +56,15 @@
   <mailingLists>
     <mailingList>
       <name>Dev Mailing List</name>
-      <post>dev@parquet.incubator.apache.org</post>
-      <subscribe>dev-subscribe@parquet.incubator.apache.org</subscribe>
-      <unsubscribe>dev-unsubscribe@parquet.incubator.apache.org</unsubscribe>
+      <post>dev@parquet.apache.org</post>
+      <subscribe>dev-subscribe@parquet.apache.org</subscribe>
+      <unsubscribe>dev-unsubscribe@parquet.apache.org</unsubscribe>
     </mailingList>
     <mailingList>
       <name>Commits Mailing List</name>
-      <post>commits@parquet.incubator.apache.org</post>
-      <subscribe>commits-subscribe@parquet.incubator.apache.org</subscribe>
-      <unsubscribe>commits-unsubscribe@parquet.incubator.apache.org</unsubscribe>
+      <post>commits@parquet.apache.org</post>
+      <subscribe>commits-subscribe@parquet.apache.org</subscribe>
+      <unsubscribe>commits-unsubscribe@parquet.apache.org</unsubscribe>
     </mailingList>
   </mailingLists>
 
-- 
1.7.9.5

